.maintain/config/bridge-pangoro-goerli.toml

```diff
@@ -1,3 +1,21 @@
+[general]
+# Whether to relay beacon headers from goerli to pangoro
+header_goerli_to_pangoro = true
```

Comment:
For easy abstraction of e2e bridge code (#511), I think "goerli" rename to "beacon " and "pangoro" rename to "substrate", but don't worry about it for now